cmd/commands/service.go

```diff
@@ -418,6 +426,8 @@ Additional curl arguments and flags may be specified after a double dash (--).`,
 	LabelArgs(command, "SERVICE_NAME", "PATH")
 
 	command.Flags().StringVarP(&serviceInvokeOptions.Namespace, "namespace", "n", "", "the `namespace` of the service")
+	command.Flags().BoolVar(&serviceInvokeOptions.ContentTypeJson, "json", false, "set the request's content type to `application/json`")
```

Comment:
Use of backticks in flags usage is for naming the type of the flag. See https://github.com/projectriff/riff/pull/787/files#diff-d631fc29cfe09b0098260aa1a1e0ede2R28. It's ok to use backticks in command usage though (with moderation and consistency as it has Markdown impacts. See cdf25c12a5b0945c00ef56302d2f52084feaab97)
Will change to regular ticks on merge